foxai/explainer/computer_vision/object_detection/types.py

```diff
@@ -11,7 +11,7 @@
 class PredictionOutput:
     """Data class for model prediction output in YOLO style."""
 
-    bbox: List[int]
+    bbox: Tuple[int, int, int, int]
```

Comment:
Haven't you just added BboxType? Below

Reply:
Fixed: https://github.com/softwaremill/FoXAI/pull/101/commits/eb61017843653e0cfede6da428c13587a28eab7f.